params/params_tmpl_am.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 2.51
+    
+# e.g. "anas_krystallinus"    
+species.name <- " Alekhya's hawk "
+    
+# e.g. "red"    
+color <- " azure4 "
```

Comment:
Similar here, the spaces might cause problems

```suggestion
color <- "azure4"
```